bounties_api/user/views.py

```diff
@@ -87,6 +90,21 @@ def get(self, request, public_address):
 
 
 class UserProfile(APIView):
+    def post(self, request, public_address):
+        user = request.current_user
+        user.is_profile_image_dirty = request.data.get('profileDirectoryHash') != str(user.profileDirectoryHash)
```

Comment:
If this is the same, does that mean the rest of this function essentially does nothing? If so we might be able to just return out of the function at that point. 

In terms of notifications and toasts, it might make sense with a "No changes found" type of message when updating without any changes - although I suppose the save/update button in the front end would be disabled if there were no changes 🤔